src/content/docs/codestream/codestream-integrations/notifications.mdx

```diff
@@ -1,55 +1,62 @@
 ---
-title: Notifications
+title: CodeStream notifications
 description: "How to get CodeStream notifications."
 ---
 
-CodeStream will notify you about comments, issues and feedback requests that you
-"follow", and you can choose whether you want to be notified via email, desktop
-(for the VS Code and JetBrains extensions only), or both. Look for the
-Notifications option under the headshot menu at the top of the CodeStream pane.
+New relic CodeStream will notify you about comments, issues, and feedback requests that you
+follow, and you can choose whether you want to be notified via email, desktop
+(for the VS Code and JetBrains extensions only), or both. Look for the notifications option under the headshot menu at the top of the CodeStream pane.
```

Comment:
I've never heard of a headshot menu. Do you mean avatar?

Reply:
I think profile menu is probably better here.

Reply:
It's where you can add a profile image (or headshot). It defaults to your initials.